backend/internal/models/game.go

```diff
@@ -4,22 +4,28 @@ import "time"
 
 type Game struct {
 	DbModel
-	SeasonID        uint
-	Teams           []Team    `json:"teams" gorm:"many2many:games_teams;"`
-	Team1LockerRoom string    `json:"home_locker_room"`
-	Team2LockerRoom string    `json:"away_locker_room"`
-	Start           time.Time `json:"start"`
-	End             time.Time `json:"end"`
-	Venue           Venue     `json:"venue"`
-	VenueID         uint
-
-	// ManagerOnCall    User      `json:"manager_on_call"`
-	// ManagerOnCallID  uint
-	// ScoreKeeper      User `json:"score_keeper"`
-	// PrimaryReferee   User `json:"primary_referee"`
-	// SecondaryReferee User `json:"secondary_referee"`
-
-	// TODO Figure out scorekeeping and game stats, I think score is just one of the game stats
-	// HomeScore int `json:"home_score"`
-	// AwayScore int `json:"away_score"`//
+	AwayTeam            Team      `json:"away_team"`
+	AwayTeamID          uint      `json:"away_team_id"`
+	AwayTeamLockerRoom  string    `json:"away_team_locker_room"`
+	AwayTeamShotsOnGoal uint      `json:"away_team_shots_on_goal"`
+	End                 time.Time `json:"end"`
```

Comment:
When are we using time.Time vs uint? We need to be consistent on when we're counting time in minutes or seconds vs time.Time

Reply:
The Duration for Penalty or PenaltyTypes are two places, which are duration in seconds. But if these aren't going to be used for calculations, and are only for display purposes then time.Time makes sense